www-src/app/lib/translation.coffee

```diff
@@ -0,0 +1,23 @@
+Polyglot = require 'node-polyglot'
+
+module.exports = class Translation
+    DEFAULT_LANGUAGE: 'en'
+
+    constructor: ->
+        @polyglot = new Polyglot()
+
+    initialize: (locale) ->
+        [@language] = locale.value.split '-'
+        @_getTranslationsFromFile()
+
+        @polyglot.t.bind @polyglot
```

Comment:
@nono I don't know but for now it's here: 
- https://github.com/cozy/cozy-mobile/blob/development/www-src/app/application.coffee#L35
- https://github.com/cozy/cozy-mobile/blob/development/www-src/app/service/service.coffee#L40